winui/Shimmer/Customization.md

```diff
@@ -0,0 +1,119 @@
+---
+layout: post
+title: Customization in WinUI Shimmer control | Syncfusion
+description: Learn how to customize the appearance of the Syncfusion WinUI Shimmer(SfShimmer) control using various properties like Fill, Wave Color, Wave Width, Repeat Count and Wave Duration.
+platform: WinUI
```

Comment:
@GilbertmanojSF3971 using various properties like Fill, WaveColor, WaveWidth, RepeatCount and WaveDuration.

Reply:
@Manivannan-E Sir Changed the content like using various properties like Fill, WaveColor, WaveWidth, RepeatCount and WaveDuration this.